ladybug_comfort/wbgt.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+"""Utility functions for calculating the Wet Bulb Globe Temperature (WBGT)."""
+from __future__ import division
+
+import math
+
+
+def wbgt_outdoors(ta, mrt, ws, vp):
+    """
+    The wet-bulb globe temperature (WBGT) is a type of apparent temperature,
+    and is widely used heat stress index globally (ISO 7243). It is used to
+    estimate the effect of temperature, humidity, wind speed (wind chill), and
+    visible and infrared radiation (usually sunlight) on humans.
+
+    WBGT outdoor formula is based on a mathematical model published by
+    K. Blazejczyk et al. in 2014 [1] with the natural wet bulb temparature
+    (Tnwb) based on the code written by Nick Burns [2].
+
+    Note:
+        [1] Heat stress and occupational health and safety spatial
+        and temporal differentiation, K. Blazejczyk, J.Baranowski, A. Blazejczyk,
+        Miscellanea geographica regional studies on development, Vol. 18,
+        No. 1, 2014
+        [2] https://github.com/nickb-/Calculating-WBGT/blob/master/relaxation_Tw/
+        working_code/wbgt.R
+
+    Args:
+        ta: Air temperature [C]
+        mrt: Radiant temperature [C]
+        ws: Wind speed [m/s]
+        vp: Vapour pressure [hPa]
+
+    Returns:
+        outdoor WBGT
+
+    """
+    # Natural wet bulb temperature
+    tnwb = 1.885 + 0.3704 * ta + 0.4492 * vp
```

Comment:
the formula I found in the [paper](https://content.sciendo.com/configurable/contentpage/journals$002fmgrsd$002f18$002f1$002farticle-p61.xml) [1] is 
`Tnw = -9.27522 + 0.70196 * Ta + 0.30338 * vp + 0.07823 * RH`
Maybe either formulas works but I just don't know, so I thought I would bring this discrepancy up and see if  anyone has ideas?

Reply:
FYI, ladybug core is a dependency of ladybug-comfort and so you can just access the `saturated_vapor_pressure` method by putting the following at the top of the file:
```
from ladybug.psychrometrics import saturated_vapor_pressure 
```